src/EFCore.PG/Infrastructure/Internal/NpgsqlOptionsExtension.cs

```diff
@@ -329,6 +350,28 @@ public class UserRangeDefinition
             SubtypeName = subtypeName;
         }
 
+        public override int GetHashCode()
+        {
+            unchecked
+            {
+                var hashCode = RangeName.GetHashCode();
+                hashCode = (hashCode * 397) ^ (SchemaName != null ? SchemaName.GetHashCode() : 0);
+                hashCode = (hashCode * 397) ^ SubtypeClrType.GetHashCode();
+                hashCode = (hashCode * 397) ^ (SubtypeName != null ? SubtypeName.GetHashCode() : 0);
```

Comment:
The same here. Otherwise, simplify conditional expressions using null-coalescing.

Reply:
Yep, missed those.

Reply:
This is resolved and the next comment too, but I see no changes. Did you forget to push changes?

Reply:
Oops, sorry about that... Now I definitely pushed the changes.